galaxy/wrapper/plotPCA.xml

```diff
@@ -78,7 +78,7 @@
             <param name="outFileFormat" value="png" />
             <param name="outFileNameData" value="True" />
             <output name="outFileName" file="plotPCA_result2.png" ftype="png" compare="sim_size" delta="12000" />
-            <output name="output_outFileNameData" file="plotPCA_result2.tabular" ftype="tabular" />
+            <output name="output_outFileNameData" file="plotPCA_result2.tabular" ftype="tabular" compare="sim_size" delta="12000" />
```

Comment:
this is not good, why is the difference so high? The idea of such tests is that we spot errors because the output changes dramatically. If the output is actually intended to change than the file `plotPCA_result2.tabular` should be updated to reflect the new "truth".

Reply:
It is indeed high. plotPCA test in planemo generates 2.49319462166397e-32 for pc2 instead of 1.2325951644078315e-32 in `plotPCA_result2.tabular` . I should not change this though, since it is actually correct. This is actually what the cmd version of deeptools generates too. What is your suggestion?

Reply:
You can check the command that planemo creates. Planemo uses the same version as the CLI version, so either the arguments are different or something strange happens here.

Reply:
I would say the latter. Maybe some difference in the version of some package or so. 
Though, let me check the planemo args one more time, in case I over looked something.